data/pools/transactionPool.go

```diff
@@ -875,6 +875,7 @@ func (pool *TransactionPool) AssembleBlock(round basics.Round, deadline time.Tim
 				stats.AverageFee = totalFees / uint64(stats.IncludedCount)
 			}
 			stats.StateProofNextRound = uint64(assembled.Block().StateProofTracking[protocol.StateProofBasic].StateProofNextRound)
+			stats.BlockHash = assembled.Block().Digest().String()
```

Comment:
I would expect this hash exists elsewhere already. Can we avoid rehashing it just to report it?

Reply:
This ended up to be too early to compute the block hash since the header does not have seed yet

Reply:
so this computes a sort of pseudo hash in which the header seed is 0?